package.json

```diff
@@ -1,22 +1,18 @@
 {
-  "name": "chat-app",
-  "version": "1.0.0",
-  "description": "",
+  "name": "trivy-scans-test",
+  "version": "0.0.1",
+  "description": "Just a test",
   "main": "index.js",
   "scripts": {
-    "start": "node src/index.js",
-    "dev": "nodemon src/index.js"
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "",
+  "author": "lior",
   "license": "ISC",
+  "repository": "Some repo string",
   "dependencies": {
-    "bad-words": "^3.0.0",
-    "express": "^4.17.1",
-    "socket.io": "^2.4.0"
-  },
-  "devDependencies": {
-    "nodemon": "^1.18.7"
+    "express": "4.18.1",
+    "lodash": "4.17.21",
+    "react": "18.2.0",
+    "axios": "0.19.2"
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-0155   
**Check Name:** follow-redirects: Exposure of Private Personal Information to an Unauthorized Actor   
**Severity:** HIGH   
**Fixed Version:** 1.14.7   
**Description:** follow-redirects is vulnerable to Exposure of Private Personal Information to an Unauthorized Actor
(This package is used under: axios->follow-redirects)  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-0155